packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -63,21 +70,56 @@ test.describe('Chat Composite E2E Tests', () => {
     await browser.close();
   });
 
-  test('composite loads correctly with participant list', async () => {
-    await pages[0].bringToFront();
-    await pages[0].waitForSelector(dataUiId(IDS.sendboxTextfield));
-    await pages[0].waitForSelector(dataUiId(IDS.participantList));
-    stubTimestamps(pages[0]);
-    expect(await pages[0].screenshot()).toMatchSnapshot('1-chat-screen.png');
+  test('composite pages load completely', async () => {
+    for (const idx in pages) {
+      await compositeLoaded(pages[idx]);
+      expect(await pages[idx].screenshot()).toMatchSnapshot(`1-page-${idx}-chat-screen.png`);
```

Comment:
What determines the names of these snapshots?

Let's avoid using numbers in the file names. When we reorder the tests in this file (or add one in the middle), we'll either end up with non-contiguous numbers while reading the file (confusing) or will have to rename all the files (unnecessary).

Reply:
Right. I named them such for local development. Makes sense to remove the numbers. Will do

Reply:
Right. I did this for local debugging. Makes sense to remove numbers.